### ISSUE TITLE
Crash! when adding pics  to the form view 

### ISSUE THREAD

**Issue by mz0in** [teableio/teable#660]
When adding a jpg or png to the form the form is and link is completely crashed 
( Crash! This view is broken. If the refresh still fails, run the data repair program. )

**Comment by tea-artist** [comment]
Upload an image in the attachment field of the shared form view? I didn't reproduce the problem

**Comment by mz0in** [comment]
not in the attachment , in the form title and main image 

<img width="814" alt="image" src="https://github.com/teableio/teable/assets/5844695/9e1ce322-4ce4-4a77-baf1-70d2564c01bf">

**Comment by Sky-FE** [comment]
Could you kindly check your Docker image version? If it's not the latest, feel free to pull the latest image and give it a spin.

**Comment by nicholas-mn** [comment]
I am on the latest docker version and getting the same problem when adding a picture to the form.

![image](https://github.com/teableio/teable/assets/9296369/3d07459d-5f8c-4e83-a207-dabc16cc8066)

I don't know if it helps debugging, but I see this error in the browser console when I try to open the now broken form view:

```
TypeError: Failed to construct 'URL': Invalid URL
    at 6508-0a5173c1a78421a5.js:1:6163
    at K (6508-0a5173c1a78421a5.js:1:10476)
    at lk (framework-922e6b3f191606ca.js:25:60903)
    at i (framework-922e6b3f191606ca.js:25:119420)
    at uD (framework-922e6b3f191606ca.js:25:99073)
    at framework-922e6b3f191606ca.js:25:98940
    at uI (framework-922e6b3f191606ca.js:25:98947)
    at uE (framework-922e6b3f191606ca.js:25:95699)
    at ux (framework-922e6b3f191606ca.js:25:94254)
    at x (framework-922e6b3f191606ca.js:49:1364)
```

**Comment by mz0in** [comment]
I'm building from the current GitHub repo and the crash as @mini-mini-oss attached

**Comment by mz0in** [comment]
now with the latest update the crash bug is solved , but no image is displayed 


<img width="936" alt="image" src="https://github.com/teableio/teable/assets/5844695/404628c3-04d6-473c-afef-b686eb719479">

**Comment by tea-artist** [comment]
> now with the latest update the crash bug is solved , but no image is displayed 
> 
> 
> 
> 
> 
> <img width="936" alt="image" src="https://github.com/teableio/teable/assets/5844695/404628c3-04d6-473c-afef-b686eb719479">
> 
> 

how do you config the storage related env, do you enabled minio?

**Comment by mz0in** [comment]
I enabled local storage using env

**Comment by tea-artist** [comment]
We have identified a similar issue and fixed it in #715 . You can try upgrading to the latest version.

**Comment by nicholas-mn** [comment]
> We have identified a similar issue and fixed it in #715 . You can try upgrading to the latest version.

Now it works, but you can not delete any of the pictures without having to delete the entire form view and start anew. You can only replace, which might not be the desired behavior all the time.
